README.md

```diff
@@ -2,50 +2,46 @@
 
-## Hack Night Meetups
+## Teams and Repositories
 
-### September 6
+The project is composed of four teams with their own corresponding repositories:
 
-On Tuesday September 6, we're [meeting up](https://www.meetup.com/DataPhilly/events/233403039/ "DataPhilly Meetup Details") at CandiDate (Floor 17 of [230 S Broad Street](https://goo.gl/maps/Anoo4SUE9At "Industrious on Google Maps")) from 6:00–8:30 PM.
+| Team Name | Repositories | Description |
+|-----------|--------------|-------------|
+| Cancer Data | [`cancer-data`](https://github.com/cognoma/cancer-data) | processing the underlying cancer data to the formats required for this project. See . |
+| Machine Learning | [`machine-learning`](https://github.com/cognoma/machine-learning) | building classifiers to predict mutation status from gene expression data. |
+| Backend | [`django-cognoma`](https://github.com/cognoma/django-cognoma), [`task-service`](https://github.com/cognoma/task-service) | creating the infrastructure to power the webapp and glue the components together. |
+| Frontend | [`frontend`](https://github.com/cognoma/frontend), [`cognoma.github.io`](https://github.com/cognoma/cognoma.github.io) | building the webapp that users interact with. |
 
-### August 23
+## New Here?
 
-On Tuesday August 23, we had a [meetup](https://www.meetup.com/DataPhilly/events/233403001/ "DataPhilly Meetup Details"). Refer to specific Project Cognoma [repositories](https://github.com/cognoma) for details.
+If you are a new user and would like to get involved, please [introduce yourself](https://github.com/cognoma/cognoma/issues/2 "Issue #2: Introduce yourself here to get involved"). Contributions are made through GitHub, so if you are unfamiliar with git or GitHub, check out the [`sandbox`](https://github.com/cognoma/sandbox) for a place to learn by doing.
 
-### August 9
+## Meetup Schedule
 
-On Tuesday August 9, we had a [meetup](https://www.meetup.com/DataPhilly/events/233070705/ "DataPhilly Meetup Details"). For those still learning how to contribute on GitHub, check out @stephenshank's [tutorial](https://youtu.be/oKlW_dSUrb8 "YouTube Video by Stephen Shank: DataPhilly/Cognoma - Submitting a pull request using Git and Github") and accompanying [pull request](https://github.com/cognoma/cognoma.github.io/pull/1 "cognoma/cognoma.github.io#1: Pull request to add favicon"). We're still looking for more community involvement in reviewing open pull requests.
+We hold project meetups. Our usual meeting spot is at Industrious (where CandiDate is located). The address is [230 S Broad St, Floor 17, Philadelphia](https://goo.gl/maps/Anoo4SUE9At "Google Maps").
 
-### July 26
+| 📅 Date | ⌚ Time | 🗺 Location | ℹ️ Meetup Details | 💰 Sponsor |
```

Comment:
Is there any reason why you don't persist the hyperlinks for CandiDate, IBI, or Milkboy?